_spinnaker_install_admin_guides/architecture.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+order: 20
+redirect_from:
+  - /spinnaker_install_admin_guides/architecture/
```

Comment:
Could anything be coming from `/spinnaker-install-admin-guides/...`?

Reply:
This is basically a redirect to handle the underscored version of the page (probably not necessary but won't hurt).